evalml/objectives/objective_base.py

```diff
@@ -76,11 +76,15 @@ def _standardize_input_type(input_data):
         """Standardize input to pandas for scoring.
 
         Arguments:
-            input_data (ww.DataTable, ww.DataColumn, pd.DataFrame, pd.Series, or np.ndarray): A matrix of predictions or predicted probabilities
+            input_data (list, ww.DataTable, ww.DataColumn, pd.DataFrame, pd.Series, or np.ndarray): A matrix of predictions or predicted probabilities
 
         Returns:
             pd.DataFrame or pd.Series: a pd.Series, or pd.DataFrame object if predicted probabilities were provided.
         """
+        if isinstance(input_data, list):
+            if isinstance(input_data[0], list):
+                return pd.DataFrame(input_data)
+            return pd.Series(input_data)
```

Comment:
Looks good.

Some suggestions:
* Replace the final `return pd.DataFrame` at the bottom with an exception and add an `if` clause for np array input, since I think that's the only supported type we don't have an `if` for yet.
* Style nit-pick: move this code below the pandas/woodwork checks, just for readability because those `if`s are shorter 🤷‍♂️